helm/prometheus-operator-app/values.yaml

```diff
@@ -35,29 +35,32 @@ prometheus-operator-app:
       ingressClassName: nginx
       pathType: ImplementationSpecific
   prometheusOperator:
-    enabled: true
-    image:
-      repository: docker.io/giantswarm/prometheus-operator
-    prometheusDefaultBaseImage: docker.io/giantswarm/prometheus
-    resources:
-      limits:
-        cpu: 200m
-        memory: 200Mi
-      requests:
-        cpu: 100m
-        memory: 100Mi
     admissionWebhooks:
-      enabled: true
       patch:
-        enabled: true
         image:
-          repository: docker.io/giantswarm/kube-webhook-certgen
-          sha: ""
-      certManager:
-        enabled: false
+          # We have 2 images for this: https://github.com/giantswarm/retagger/blob/dc815cbec228a32d7dc1859e6e3c73bef5488b52/images.yaml#L1069 and https://github.com/giantswarm/retagger/blob/dc815cbec228a32d7dc1859e6e3c73bef5488b52/images.yaml#L513
+          # The jettech one is old and does not work with Kubernetes 1.22. We need to deprecate it later.
+          repository: docker.io/giantswarm/ingress-nginx-kube-webhook-certgen
+    alertmanagerDefaultBaseImage: docker.io/giantswarm/prometheus
+    image:
+      repository: docker.io/giantswarm/prometheus-operator
```

Comment:
How does those images work in china? We need to be able to change the registry there.

Reply:
This is already set in 2.0.0

Reply:
Can we open an issue to add registry support upstream and create a new version of the app?

Reply:
I upgraded to 42.0.2 so we can override the images now :)

Reply:
Not sure how it goes with WC in China though